requirements.txt

```diff
@@ -9,7 +9,9 @@ brotlipy
 pyyaml
 werkzeug
 webencodings
-gevent==1.2.2
+dnspython     # for updated gevent: recommend by gevent, c-ares resolver to be deprecated soon
+idna          # for updated gevent: Python 2.7 support
```

Comment:
It looks like `idna` is already required by `requests` so probably don't need to include it explicitly

Reply:
done